auth.config.ts

```diff
@@ -0,0 +1,13 @@
+import path from 'path';
+export const authConfig = {
+  admin: {
+    username: "Admin",
+    password: "admin123",
+    storageState: path.join(process.cwd(), 'playwright/.auth/admin.json')
+  },
+  staff: {
+    username: "Admint",
```

Comment:
should we use username/password of staff with staff name instead of admin? To avoid any confusion

Reply:
Done